### DISCUSSION TITLE
special context for other sciences

### DISCUSSION THREAD

**Discussion by kalle07** [BaranziniLab/KG_RAG#16] [Ideas]
you have the intention to use other than SPOKE  for bio ?
maybe more social or mor technical ?

**Comment by karthiksoman** [comment]
Hi @kalle07 
As a framework, KG-RAG is general enough to apply on domains other than biomedicine. From our application standpoint, we intend to use KG-RAG for biomedical purposes using SPOKE as the backend graph. But, happy to discuss this more if you have ideas on implementing this on other domains too.

**Comment by kalle07** [comment]
i see ...
ok, can i easy implement it in text-generation-webui - oobadooga  ?
so if not ... its for most people not interessting and not worth the effort    ;)

**Comment by karthiksoman** [comment]
@kalle07 
'text-generation-webui' is used for having a user interface for the LLM. I am not sure if 'text-generation-webui' supports rag framework (not just kg-rag, any rag framework). if you are aware of any such support offered by 'text-generation-webui', let me know.

**Comment by kalle07** [comment]
iam only a stupied user, ok i can type 
"create conda" 
"git clone" and 
"pip install ..." 

so its 99% more than all users in the www ^^

so its on you to prepare a compatibility ^^
talk with oobadooga  :)

**Comment by karthiksoman** [comment]
@kalle07 
Thanks for your suggestion. I can checkout for the user interface options available for rag frameworks and also feel free to let me know if you hit with any such options.
